src/main/server.js

```diff
@@ -33,7 +33,11 @@ module.exports = function(options) {
   app.use(cookieParser());
   app.use(express.static(options.publicDir));
   // App.use(require("../index").app);
-
+if(options.routes) {
```

Comment:
please add a check for array and checks for if the files have module exports and that it is a router, use throw TypeError("descriptive message")

Reply:
Yes